api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -19,34 +17,27 @@
 public class InstitutionServiceImpl implements InstitutionService {
 
   private final InstitutionDao institutionDao;
-  private final InstitutionEmailDomainDao institutionEmailDomainDao;
-  private final InstitutionEmailAddressDao institutionEmailAddressDao;
 
   @Autowired
-  InstitutionServiceImpl(
-      InstitutionDao institutionDao,
-      InstitutionEmailDomainDao institutionEmailDomainDao,
-      InstitutionEmailAddressDao institutionEmailAddressDao) {
+  InstitutionServiceImpl(InstitutionDao institutionDao) {
     this.institutionDao = institutionDao;
-    this.institutionEmailDomainDao = institutionEmailDomainDao;
-    this.institutionEmailAddressDao = institutionEmailAddressDao;
   }
 
   @Override
   public List<Institution> getInstitutions() {
     return StreamSupport.stream(institutionDao.findAll().spliterator(), false)
```

Comment:
I'd probably put this on the dao itself as a `default List<Institution> getAllInstitutions()` so all services using it get the benefit.

Reply:
Should our DAO level be aware of our API?

Reply:
You mean of the type? Good point. In practice there's not much cyclical dependence added but you're right.

This is why I think we need a DataManager layer.